examples/getting_started_ms.py

```diff
@@ -0,0 +1,31 @@
+from mseetc.train import Train
```

Comment:
Please revert this change and place this code back in ocp.py, as this re-structuring is not relevant for the PR.